Dockerfile

```diff
@@ -27,8 +27,9 @@ RUN sudo apt-get update && sudo apt-get -y install ros-kinetic-ros-base
 RUN echo "source /opt/ros/kinetic/setup.bash" >> ~/.bashrc
 
 # Get the erdos directory.
-RUN mkdir -p /home/erdos/workspace/erdos
-ADD . /home/erdos/workspace/erdos
+RUN sudo apt-get -y install git
+RUN mkdir -p /home/erdos/workspace
+RUN cd /home/erdos/workspace && git clone https://github.com/erdos-project/erdos.git
 RUN echo "export PYTHONPATH=\$PYTHONPATH:/home/erdos/workspace/erdos" >> ~/.bashrc
 WORKDIR /home/erdos/workspace/erdos
```

Comment:
Can you please invoke install_dependencies.sh in this Dockerfile? At the moment we duplicate the logic both.

Reply:
The Dockerfile now invokes install_dependencies.sh.